typedspark/_schema/get_schema_definition.py

```diff
@@ -64,6 +66,25 @@ def _build_schema_definition_string(
     return lines
 
 
+def _replace_literal_by_intervaltype(typehint, literal, intervaltype):
```

Comment:
This new function and the one below would be good candidates for a unit test

Reply:
Good suggestion, done!